distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1985,40 +1770,48 @@ public void testCancelDownload() throws Exception {
                 RESOURCE_DOWNLOAD_ENDPOINT_ROOT.getUrl() + "?source=" + CSW_STUB_SOURCE_ID
                         + "&metacard=" + metacardId;
 
-        // @formatter:off
-        String downloadId = given().auth().preemptive().basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
-                .get(startDownloadUrl).then().log().all()
-                .extract().jsonPath().getString("downloadId");
-        // @formatter:on
+        given().auth()
+                .preemptive()
+                .basic(LOCALHOST_USERNAME, LOCALHOST_PASSWORD)
+                .get(startDownloadUrl)
+                .then()
+                .log()
+                .all();
 
-        localhostCometDClient.cancelDownload(downloadId);
+        expect("Waiting for download to start.").within(30, SECONDS)
+                .until(() -> localhostCometDClient.getDownloadIds()
+                        .size() > 0);
 
-        expect("Waiting for notifications.").within(10, SECONDS)
-                .until(() -> localhostCometDClient.getMessages(NOTIFICATIONS_CHANNEL)
-                        .size() == 1);
+        assertThat(localhostCometDClient.getDownloadIds()
```

Comment:
@coyotesqrl, you have a good point. Our CSW stub server and `ChunkContent` class don't allow the test code to pause and restart the data being sent back. The best it can do right now is to return the data slowly, in this case one character every 200ms. Given the size of the response (58 characters) and the delay between bytes, it will take almost 12s for the data to be completely sent and the download marked as completed. If we up the delay to 300ms, we're looking at over 17s for the download to fully complete.

I think it would make a lot of sense to add that capability to the `ChunkContent` class. However, given the extremely unlikely case where all the data would be sent by the time we start waiting on the download ID, would you be ok with us increasing the delay to 300ms (or even 400ms) and writing up a separate ticket to improve the `ChunkContent` class to support pause/resume? Adding that in will probably require a fair amount of refactoring and we'd like to get these tests re-enabled soon.

@bantillo, we should remove that comment.